pkg/inspect/inspect.go

```diff
@@ -14,14 +15,25 @@ const (
 	REALITY_SOURCE
 )
 
+type LaunchableVersion struct {
+	Location string                    `json:"location,omitempty"`
```

Comment:
maybe there is a case for adding a `location:` option to `LaunchableVersion` though, so that there's only one place to specify version